src-java/kilda-model/src/main/java/org/openkilda/model/Metadata.java

```diff
@@ -15,37 +15,46 @@
 
 package org.openkilda.model;
 
-import lombok.Value;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
 
 import java.io.Serializable;
 
 /**
  * Represents information about a metadata.
- * Uses 64 bit to encode information about the packet:
- *  0                   1                   2                   3
- *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ * Uses 32 bit to encode information about the packet:
+ * 0                   1                   2                   3
+ * 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- * |L|O|                      Reserved Prefix                      |
- * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Please do not remove space before zero. 

Before removal it was easy to understand that `L` is in `0` bit, `O` is in 1 bit. After removing it's hard to understand that.

especially in the middle. Digits are under `|` now
```
4 5 6 7 8 
+-+-+-+-+
|F|O|L|A|
```